torch/csrc/jit/frontend/builtin_functions.cpp

```diff
@@ -49,6 +49,12 @@ def ndim(a : Tensor) -> int:
   return a.dim()
 def T(a : Tensor) -> Tensor:
   return a.numpy_T()
+def H(a : Tensor) -> Tensor:
+  return a.matrix_H()
+def mT(a : Tensor) -> Tensor:
```

Comment:
these aren't tensor attributes right? at the least, these shouldn't be added here

Reply:
I thought they were properties? As per the PR title?